src/VerifyAccountHandler.ts

```diff
@@ -35,14 +36,23 @@ export class VerifyAccountHandler implements LambdaInterface {
 		try {
 			const { sessionId, body, encodedHeader } = this.validateEvent(event);
 			const clientIpAddress = event.headers[Constants.X_FORWARDED_FOR] ?? event.requestContext.identity?.sourceIp;
+			const credentialVendorSsmPath = checkEnvironmentVariable(EnvironmentVariables.CREDENTIAL_VENDOR_SSM_PATH, logger);
+			CREDENTIAL_VENDOR = await getParameter(credentialVendorSsmPath);
+			if (CREDENTIAL_VENDOR === "HMRC") {
+				const hmrcTokenSsmPath = checkEnvironmentVariable(EnvironmentVariables.HMRC_TOKEN_SSM_PATH, logger);
+    			VENDOR_TOKEN = await getParameter(hmrcTokenSsmPath);
+
+				logger.appendKeys({ sessionId });
+				logger.info("Starting VerifyAccountRequestProcessorHmrc");
+				return await VerifyAccountRequestProcessor.getInstance(logger, metrics, VENDOR_TOKEN).processHmrcRequest(sessionId, body, clientIpAddress, encodedHeader);
+			} else {
+				const experianTokenSsmPath = checkEnvironmentVariable(EnvironmentVariables.EXPERIAN_TOKEN_SSM_PATH, logger);
+				VENDOR_TOKEN = await getParameter(experianTokenSsmPath);
```

Comment:
Will this be replaced by the token table?

Reply:
I'll check with Stelios

Reply:
@Stelios-CB